sycl/doc/extensions/experimental/sycl_ext_oneapi_bindless_images.asciidoc

```diff
@@ -1396,9 +1396,15 @@ As with allocation, the descriptor must be populated appropriately, i.e.
 
 === Copying image array data [[copying_image_array_data]]
 
-When copying to or from image arrays, the user should copy to/from the entire 
-array of images in one call to `ext_oneapi_copy` by passing the image arrays'
-`image_mem_handle`.
+When copying to or from image arrays, the user should copy to/from the array of
+images by calling `ext_oneapi_copy` and passing the image arrays'
+`image_mem_handle`, as outlined in <<explicit_copies>>.
+
+[NOTE]
+====
+When performing sub-region copies involving image arrays, the offset and extent
+for the layers is always the 3rd dimension of the ranges passed.
+====
```

Comment:
Looking at this again. I think it might make more sense to have it as a real paragraph and not within a `[NOTE]`.

I would also prefer for it to be more explicit. E.g.

```suggestion
[NOTE]
====
When performing sub-region copies involving image arrays, the source and destination offsets 
are always the third dimension of the source and destination extents respectively, and the layer count to be copied is always the passed through the third dimention of the copy extent parameter. This holds for both one- and two-dimensional image arrays.
====
```

Reply:
I've changed this to be more explicit.